WFInfo/Data.cs

```diff
@@ -1103,7 +1101,7 @@ public async Task<bool> OpenWebSocket()
             Main.AddLog("Connecting to websocket");
             marketSocket.SslConfiguration.EnabledSslProtocols = SslProtocols.Tls12;
 
-            if (marketSocket.IsAlive || marketSocket.ReadyState == WebSocketState.Connecting)
+            if (marketSocket.IsAlive) //|| marketSocket.ReadyState == WebSocketState.Connecting
```

Comment:
I suggest we remove the commented part altogether, no point to keep it there if its not needed anymore.

Reply:
+1